src/schemes.cpp

```diff
@@ -171,15 +155,15 @@ bool CoreMPL::AggregateVerify(
     const vector<uint8_t> &signature)
 {
     int n = pubkeys.size();
+    G2Element signatureElement = G2Element::FromByteVector(signature);
     if (n <= 0 || n != messages.size()) {
```

Comment:
It looks like we are only checking args if we go into this if statement. Why not always check, for consistency with the other call locations?

I think a better way to do it, is to not check in the caller location. And instead do this:

if (!VerifyAggregateSignatureArguments(n, messages.size(), signatureElement) {
    return False
}

Reply:
Makes sense 👍 I just unified it in all places in 0de8f04e19c2ce03761c1bbd1b75f44a30049e19, thoughts?